Another projekts/calculator/main_calculator_1.py

```diff
@@ -0,0 +1,31 @@
+run = True
+
+while run:
+    first_number = int(input("Enter first number: "))
+    second_number = int(input("Enter second number: "))
+    what_todo = input("What do you want to do?\n1 for +\n2 for -\n3 for *\n4 for /\n: ")
+
+    def mathematical_operation(choise):
```

Comment:
Just move this out of the while: loop.
As far as I know, it's a bad coding style that you have here (to define functions inside others).
Please brush up on your knowledge of the functions and main context.

Defining a function inside a while loop in Python is generally not considered good practice, and it can cause issues in certain situations. Here are a few reasons why:

- Performance: Defining a function inside a loop means that the function will be redefined every time the loop iterates. This can lead to unnecessary overhead and slow down the execution of the code.
- Code clarity: Defining a function inside a loop can make the code less readable and harder to understand, especially if the loop is long or complex.
- Scope: If the function is defined inside a loop, it will only be accessible within that loop. This can cause issues if you need to use the function outside of the loop or in other parts of your code.

Reply:
I exported the function to another file and imported it into the working program.
First_number and second_number i added in inside function in another file which i import to my main file.
I can move out the what_todo, but then user cant choose what to do in next iteration always will be completed the operation whose he choice in first time